polyfill/test/helpers/temporalHelpers.js

```diff
@@ -117,6 +117,30 @@ var TemporalHelpers = {
     assert.sameValue(yearMonth.monthCode, monthCode, `${description} monthCode result`);
   },
 
+  /*
+   * assertZonedDateTime(zdt, year, ..., nanosecond[, description[, era, eraYear]]): XXX
+   *
+   * Shorthand for asserting that each field of a Temporal.ZonedDateTime is
+   * equal to an expected value. (Except the `calendar` property, since callers
+   * may want to assert either object equality with an object they put in there,
+   * or the result of zdt.calendar.toString().)
```

Comment:
```suggestion
   * equal to an expected value. (Except the `calendar` and `timeZone`
   * properties, since callers may want to assert either object equality with
   * an object they put in there, or the result of zdt.calendar.toString() or
   * zdt.timeZone.toString().)
```